### PR TITLE
e2e - regression tests

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/pageObjects/my-redis-databases-page.ts [hunk 1]
```diff
@@ -90,4 +100,16 @@ export class MyRedisDatabasePage {
           }
       }
   }
+
+  async clickOnEditDBByName(databaseName: string): Promise<void>{
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/my-redis-databases-page.ts:104]: Please add a description on top (I see it didn't happen also on functions above)

File: tests/e2e/tests/regression/browser/database-overview.e2e.ts [hunk 2]
```diff
@@ -38,3 +40,13 @@ test
...
     });
+test
+    .meta({ rte: rte.standalone })
+    .after(async () => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
+    ('Verify that user can connect to DB and see breadcrumbs at the top of the application in Browser page', async t => {
+        //Verify that user can see breadcrumbs
+        await t.expect(browserPage.breadcrumbsContainer.visible).ok('User can see breadcrumbs in Browser page', { timeout: 20000 });
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/browser/database-overview.e2e.ts:51]: 1 line test? couldn't it be covered over lower levels (such as components)

File: tests/e2e/tests/regression/cli/cli.e2e.ts [hunk 3]
```diff
@@ -91,3 +97,33 @@ test
...
+    ('Verify that user can run command json.get and see JSON object with escaped quotes (\" instead of ")', async t => {
+        //Add Json key with json object
+        await browserPage.addJsonKey(keyName, keyTTL, jsonValue);
+        const command = `JSON.GET ${keyName}`;
+        //Open CLI and run command
+        await t.click(cliPage.cliExpandButton);
+        await t.typeText(cliPage.cliCommandInput, command);
+        await t.pressKey('enter');
+        //Verify result
+        await t.expect(cliPage.cliOutputResponseSuccess.innerText).eql(`"${jsonValue.replace(/"/g, '\\"')}"`, 'The user can see JSON object with escaped quotes');
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli.e2e.ts:116]: Please fix the eslint error

File: tests/e2e/pageObjects/my-redis-databases-page.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/pageObjects/my-redis-databases-page.ts:112]: done

File: tests/e2e/tests/regression/browser/database-overview.e2e.ts (hunk 2 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/browser/database-overview.e2e.ts:51]: another verification is added in this test

File: tests/e2e/tests/regression/cli/cli.e2e.ts (hunk 3 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/regression/cli/cli.e2e.ts]: done

**Review by elenaNaboko — APPROVED** [review]

**Pull request merged**